src/webapp.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+
+import sentry_sdk
+from flask import jsonify, request, Flask
+
+from .handle_event import handle_event
+
+APP_DSN = os.environ.get("APP_DSN")
+if APP_DSN:
+    # This tracks errors and performance of the app itself rather than GH workflows
+    sentry_sdk.init(
```

Comment:
The comment got lost.

This is the official way of setting up Sentry's SDK rather than what I had.